### DISCUSSION TITLE
Feature request for self-hosting: add option to easily add/remove domains

### DISCUSSION THREAD

**Discussion by zcyph** [anonaddy/anonaddy#391] [Feature Requests]
Currently the instructions to [self-host](https://anonaddy.com/self-hosting) AnonAddy entail adding domains to `/etc/postfix/mysql-virtual-alias-domains-and-subdomains.cf` and manually adding a bunch of things to the database (ie, via `sudo mysql -u root -p`) which is all well and good for an initial setup that doesn't need to be later tinkered with.

I'm talking about adding or removing the domains that would be available to all users; not the existing functionality where individual users can add custom domains to their specific account. Either having some functionality built into the UI for this or having the functionality available through an API to safely make such changes would be greatly appreciated.

**Comment by willbrowningme** [comment]
Thanks, I don't really have time to work on self-hosting specific features at the moment but hopefully can look into this in the future.
